src/map/itemdb.cpp

```diff
@@ -1259,6 +1259,392 @@ int16 ItemGroupDatabase::item_exists_pc(map_session_data *sd, uint16 group_id)
 	return -1;
 }
 
+const std::string LaphineSynthesisDatabase::getDefaultLocation(){
+	return std::string( db_path ) + "/laphine_synthesis.yml";
+}
+
+uint64 LaphineSynthesisDatabase::parseBodyNode( const YAML::Node& node ){
+	t_itemid item_id;
+
+	{
+		std::string name;
+
+		if( !this->asString( node, "Item", name ) ){
+			return 0;
+		}
+
+		std::shared_ptr<item_data> id = item_db.search_aegisname( name.c_str() );
+
+		if( id == nullptr ){
+			this->invalidWarning( node["Item"], "laphine_synthesis_parseBodyNode: Unknown item \"%s\".\n", name.c_str() );
```

Comment:
Were we starting to put the function name in the warnings now?

Reply:
Can be removed. I think I just copied and adjusted it when I originally wrote it.